### DISCUSSION TITLE
자동예약버튼이 없는데 어디서 찾아야하는지 알 수 있을까요??

### DISCUSSION THREAD

**Discussion by SHS-Technology** [changdoc/naver-vaccine-macro#141] [Q&A]
아무리찾아도 확인이 안됩니다. ㅠㅠ

**Comment by YounghuSong** [comment]
![image](https://user-images.githubusercontent.com/77137340/127425800-38ec4558-b638-4615-ab8c-86d0b8b9ddbf.png)

예약신청 페이지에 상단에 있습니다.
